site/en/docs/extensions/reference/offscreen/index.md

```diff
@@ -48,19 +48,36 @@ chrome.offscreen.createDocument({
 chrome.offscreen.closeDocument()
 ```
 
-The following example shows how to ensure that the offscreen document has already been created. The `setupOffscreenDocument()` function calls [`clients.matchAll()`](https://developer.mozilla.org/docs/Web/API/Clients/matchAll) to find the existing offscreen document or creates it if it doesn't already exist. Note that an extension can only have one offscreen document.
+The following example shows how to ensure that the offscreen document has already been created. The `setupOffscreenDocument()` function calls [`runtime.getContexts()`][runtime-get-contexts] to find the existing offscreen document or creates it if it doesn't already exist. Note that an extension can only have one offscreen document.
+
+{% Aside 'gotchas' %}
```

Comment:
This really goes beyond what notes are intended for. I would do this. Put all of the current (i.e. post 116) stuff together worded as just the way to do it. Right after that, put a next level heading that titled something like "Before Chrome 116".

Reply:
Moved into a new section as you suggested (taking this over from Oliver).